lib/experimental-default-theme.json

```diff
@@ -165,6 +165,9 @@
 			"spacing": {
 				"customPadding": false,
 				"units": [ "px", "em", "rem", "vh", "vw" ]
+			},
+			"borderStyles": {
```

Comment:
How about using `border` instead, as we do in `lib/block-supports/border.php` and follows the naming schema of the other groups (`typography`, `color`, `spacing`)?

The properties within could also remove the `border` infix and be something like `customRadius`.

Reply:
That makes more sense. Done ✅